notebooks/building_production_ml_systems/solutions/1_training_at_scale_vertex.ipynb

```diff
@@ -61,7 +61,7 @@
     "%env BUCKET=$BUCKET\n",
```

Comment:
<p>We aren't doing objective 3 in this lab anymore. We can remove it or update it to say (optional) learn how to submit a training job using the python SDK. That's the optional part of this lab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Do you mean custom container training? </p><p>I think it still remains at the end of the notebook.</p>